src/templates/commit_page.handlebars

```diff
@@ -193,14 +193,14 @@
                 <td name="maintainer"  id="maintainer_original">
                     {{#if original.maintainer}}
                         {{#if original.maintainer.github}}
-                            <a href="https://github.com/{{original.maintainer.github}}">{{original.maintainer.github}}</a>
+                            <a href="https://duckduckhack.com/u/{{slug original.maintainer.github}}">{{original.maintainer.github}}</a>
```

Comment:
@MariagraziaAlastra We don't have `https` *yet*, but I think we will after #1330!